app/models/miq_expression.rb

```diff
@@ -118,6 +119,8 @@ class MiqExpression
     ontap_raid_group_extents
     ontap_storage_systems
     ontap_storage_volumes
+    openscap_results
+    openscap_rule_results
```

Comment:
Doesn't this keep the dependency on #6839?  I saw that it's crossed out in the PR summary.